src/styles/common/_grid.scss

```diff
@@ -0,0 +1,54 @@
+$gap: 1rem;
+$max-width: 100ch;
```

Comment:
How was this number chosen?
Is there a good reason for using `ch` instead of `rem`?

Reply:
The unit was chosen because it matches best what we want to express when we decide the main content width (as in UX says "The main content should be x characters wide" -> `max-width: <x>ch;`).
The value itself I just chose a simple value when doing the prototype.
We need to discuss it with UX looking at recommendations for accessibility. We probably need to consider different main-content widths depending on the type of content (text-heavy as an entry page _vs_ other as the home page)